FlashDevelop/Bin/Debug/Projects/380 Haxe - OpenFL Project/project.xml.template

```diff
@@ -23,4 +23,6 @@
 	<!-- optimize output
 	<haxeflag name="-dce full" /> -->
 	
+	<define name="AIR_SDK" value="${AIR_SDK}"></define>
```

Comment:
```xml
<!-- Support for Adobe AIR: http://community.openfl.org/t/official-support-for-adobe-air/9468 -->
<!--<define name="AIR_SDK" value="path/to/your/air/sdk" if="air" /> -->
```
I think that true way is edit `~/.lime/config.xml` and add the following:
`<set name="AIR_SDK" value="path/to/your/air/sdk" />`